### PR TITLE
Add action for node force-machine-remove

### PR DESCRIPTION
**Pull request by jordojordo**
Reference #4538 

This adds support for force removing a RKE2 pool in a cluster that is stuck in deleting state.

https://user-images.githubusercontent.com/40806497/142700416-49c0ea41-f517-491b-973d-b794b2f15330.mp4

**To Test**

On 2.6-head commit id: 78b25c3

- Deploy an RKE2 cluster with 2 nodes
- Invalidate the cloud credentials used to provision the cluster by deleting the access token on the cloud provider side (e.g. Digital Ocean / AWS API token).
- Delete the cluster
- The cluster is stuck in deleting state
- Select `Force Delete` from action menu for a node
- The node get removed and the cluster gets deleted from rancher

### CODE REVIEW

File: models/cluster.x-k8s.io.machine.js [hunk 1]
```diff
@@ -23,10 +24,18 @@ export default class CapiMachine extends SteveModel {
       icon:       'icon icon-fw icon-download',
       label:      this.t('node.actions.downloadSSHKey'),
     };
+    const forceRemove = {
+      action:     'toggleForceRemoveModal',
+      altAction:  'forceMachineRemove',
+      enabled:    !!this.isRemoveForceable,
+      label:      'Force Delete',
```
Comment (richard-cox) [inline models/cluster.x-k8s.io.machine.js:31]: Needs i10n

File: models/cluster.x-k8s.io.machine.js [hunk 2]
```diff
@@ -49,6 +58,28 @@ export default class CapiMachine extends SteveModel {
...
+    this.$dispatch('promptModal', {
+      resources,
+      component: 'ForceMachineRemoveDialog'
+    });
+  }
+
+  async forceMachineRemove() {
+    const machine = await this.machineRef();
+
+    machine.setAnnotation('provisioning.cattle.io/force-machine-remove', 'true');
```
Comment (richard-cox) [inline models/cluster.x-k8s.io.machine.js:71]: There's a file that contains a list of annotations, this should probably be listed there (`./config/labels-annotations.js` `CAPI` obj)

**Review by richard-cox — CHANGES_REQUESTED** [review]
Two very minor comments, otherwise LGTM

**Comment by jordojordo** [comment]
@richard-cox I updated this with the suggestions you made, moved the label to translations and added an annotation for `FORCE_MACHINE_REMOVE`

**Review by richard-cox — APPROVED** [review]

**Pull request merged**